scripts/fix_merged_user_quickfiles.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import sys
+
+from django.db import transaction
+from django.db.models import F
+
+from website.app import setup_django
+setup_django()
+from osf.models import AbstractNode
+from scripts import utils as script_utils
+
+
+logger = logging.getLogger(__name__)
+
+def main():
+    dry = '--dry' in sys.argv
+    if not dry:
+        # If we're not running in dry mode log everything to a file
+        script_utils.add_file_logger(logger, __file__)
+    with transaction.atomic():
+        qs = AbstractNode.objects.filter(type='osf.quickfilesnode').exclude(_contributors=F('creator'))
```

Comment:
Drive-by: Instead of filtering on type, you should be able to use the concrete class.

```python
QuickFilesNode.exclude(...)
```

Reply:
@sloria Neat, fixed 👍